scripts/download_data.py

```diff
@@ -116,6 +116,45 @@ def getMaxPos(self, chrom):
         return minMax.maxPos
 
 
+def fetchSequence(ac, startIndex=None, endIndex=None, as_list=False):
```

Comment:
``as_list`` -> ``asList``. We need to camelCase all identifiers I'm afraid. (I'm looking forward to the change to protobuf so I won't need to say this any more!)